### ISSUE TITLE
ClassLoader cant find properties-file, when use maven-exec-plugin:exec:java

### ISSUE THREAD

**Issue by lanwen** [qatools/properties#15]
class
~~~java
public class MainExec {
    public static void main(String[] args) {
    // ...
               new Properties();
      }
~~~

pom:
~~~xml
 <plugins>
            <plugin>
                <groupId>org.codehaus.mojo</groupId>
                <artifactId>exec-maven-plugin</artifactId>
                <version>1.2.1</version>
                <executions>
                    <execution>
                        <goals>
                            <goal>java</goal>
                        </goals>
                    </execution>
                </executions>
                <configuration>
                    <mainClass>ru.yandex.jenkins.MainExec</mainClass>
                </configuration>
            </plugin>
        </plugins>
~~~

exec:
`mvn clean compile exec:java`

produce non populated props

**Comment by baev** [comment]
@lanwen could you please check this one? Should be fixed 2.0

**Comment by baev** [comment]
fixed in 2.0
